packages/server/migrations/20230912195046_add-grants-opportunity-status-index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+    return knex.raw(`CREATE INDEX IF NOT EXISTS idx_grants_opportunity_status ON grants USING btree (opportunity_status) WHERE (opportunity_status <> 'archived'::text)`);
+};
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.down = function (knex) {
+    return knex.raw('DROP INDEX IF EXISTS idx_grants_opportunity_status;');
```

Comment:
I don't think the inner `;` on the raw SQL needs to exist. I'm not sure whether it matters to Knex, but noting it because it at least seems inconsistent with the `exports.up` migration function.

Suggest removing (or else please let me know if it is in fact significant!):
```suggestion
    return knex.raw('DROP INDEX IF EXISTS idx_grants_opportunity_status');
```

Reply:
Good point, I'm not sure whether it is significant or not. Based on the screenshot I added to the PR it seems like it executes whatever we provide to the database. It doesn't seem to have any adverse impacts, but makes sense to stay consistent.